src/core/client/auth/components/SignIn.css

```diff
@@ -0,0 +1,14 @@
+.root {
+  width: 330px;
+  padding-top: calc(5 * var(--spacing-unit));
+}
+
+.forgotPassword {
+  text-align: right;
+}
+
+.footer,
+.subFooter {
```

Comment:
See above, this is obsolte, once FormField is fixed

Reply:
👍